src/puyapy-stubs/_contract.pyi

```diff
@@ -11,6 +33,7 @@ class Contract(abc.ABC):
         *,
         name: typing.LiteralString = ...,
         scratch_slots: urange | tuple[int | urange, ...] | list[int | urange] = ...,
+        state_totals: StateTotals | None = ...,
```

Comment:
Is the `None` here to allow explicitly choosing the auto calculated values when deriving from a class that specifies state_totals? If so could we make use the literal string "auto" instead?
`state_totals: StateTotals | typing.Literal["auto"] = "auto",`

Reply:
For expediency resolved in a commit myself 🙂